internal/backend/runtime/cosi/cosi.go

```diff
@@ -171,7 +171,7 @@ func WatchLegacy(ctx context.Context, st state.State, md resource.Metadata, out
 				}
 
 				if items != nil {
-					slices.Delete(items, index, index+1) //nolint:govet
+					items = slices.Delete(items, index, index+1)
```

Comment:
Omg, there should be a linter catching this

Reply:
there's but I've disabled it 🫠